terraform.tf.j2

```diff
@@ -108,3 +108,27 @@ locals {
   }
 }
 
+locals {
+
+  env_prefix = {
+    development = "dev."
+    qa          = "qa."
+    stage       = "stg."
+    integration = "int."
+    preprod     = "pre."
+    production  = ""
+  }
+
+}
+
+data "terraform_remote_state" "aws_certificate_authority" {
+  backend = "s3"
+  config = {
+    bucket         = "{{state_file_bucket}}"
+    key            = "terraform/dataworks/aws-certificate-authority.tfstate"
+    region         = "{{state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{state_file_region}}:{{state_file_account}}:key/{{state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+  }
+}
```

Comment:
Could you make sure your editor is set to save files with a trailing newline character please?